umap/static/umap/js/umap.controls.js

```diff
@@ -1076,6 +1078,9 @@ L.U.Map.include({
 
   renderShareBox: function () {
     const container = L.DomUtil.create('div', 'umap-share')
+    const title = L.DomUtil.create('h3', '', container)
+    title.textContent = L._('Share, download and embed this map')
```

Comment:
You mean 4px left, so that the icon is above the next headline?
![grafik](https://github.com/umap-project/umap/assets/903453/10db56ec-cff7-4d43-9402-c193131b8b44)